### DISCUSSION TITLE
Would love to corporate in UI design

### DISCUSSION THREAD

**Discussion by 0x7km** [PeterStrick/ViVeTool-GUI#72] [General]
Would love to corporate in UI design, designing the UI & UX for ViVeTool GUI. I'm just not an expert in programming, in case we develop some better UI what extension would like the design files would be, like CSS or something else?

**Comment by PeterStrick** [comment]
ViVeTool GUI currently uses an old Version of Telerik UI for WinForms for it's UI. Planned for Version 2.0 is to use WinUI 3 somewhat like [WingetUI does](https://github.com/marticliment/WingetUI)
